2-mandatory/5-reading-list.js

```diff
@@ -48,6 +51,15 @@ As an example for this exercise, you might do the following steps
 - All tests should turn green!!
 **/
 
+function logBooks() {
+  for(let book of books) 
+  if(book.alreadyRead === true) {
```

Comment:
Hi, your code is really neat! I had a suggestion on my code, so I am passing it along: predicates can be used without the comparison in if else statements. So book.already without the `=== true`.

Reply:
Thanks for that, I always forget i can omit true/false in this case